script.js

```diff
@@ -0,0 +1,59 @@
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function displayBooks() {
+  const booksDiv = document.getElementById('books');
+  booksDiv.innerHTML = '';
+
+  if (books.length === 0) {
+    booksDiv.textContent = 'No books in the collection.';
+    return;
+  }
+
+  const removeBook = (index) => {
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    displayBooks();
+  };
```

Comment:
- [OPTIONAL] It's always best practice to avoid mutating the original array whenever possible. By using the `filter` method, we can create a new array with the desired elements and keep the original array intact. This promotes code clarity, and maintainability, and reduces the risk of unintended side effects. Find more [info about how the `filter` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)method works.